lib/std/os/windows.zig

```diff
@@ -2005,10 +2005,6 @@ pub fn HeapDestroy(hHeap: HANDLE) void {
     assert(kernel32.HeapDestroy(hHeap) != 0);
 }
 
-pub fn LocalFree(hMem: HLOCAL) void {
-    assert(kernel32.LocalFree(hMem) == null);
-}
-
```

Comment:
this breaking change should be a separate PR. Your PRs require a lot of reviews, so please limit yourself to 1 open PR at a time. you're already 5 over this quota.